app/views/events/_table_badges.html.erb

```diff
@@ -0,0 +1,30 @@
+<span class="badge"><%= table.scenario.tier %></span>
```

Comment:
Good to have this file!
I'm spoiled by angular having fancy looking ways for doing if/elses though. There's no syntax sugar to reduce the number of <% %> I presume.